apps/web/cypress/e2e/multi-dwelling/multi-dwelling-9.10.14-walkthrough.cy.ts

```diff
@@ -24,6 +29,19 @@ describe("multi dwelling: 9.10.14 walkthrough", () => {
     });
   });
 
+  it("should display all parts of breadcrumbs", () => {
+    cy.getByTestID(TESTID_BREADCRUMBS).should("be.visible");
+    cy.getByTestID(
+      GET_TESTID_BREADCRUMBS_BREADCRUMB(EnumBreadcrumbIds.HOME),
+    ).should("be.visible");
+    cy.getByTestID(
+      GET_TESTID_BREADCRUMBS_BREADCRUMB(EnumBreadcrumbIds.MULTI_DWELLING_UNIT),
+    ).should("be.visible");
+    cy.getByTestID(
+      GET_TESTID_BREADCRUMBS_BREADCRUMB(EnumBreadcrumbIds.LAST),
```

Comment:
I gave the last one specific unique ID since it was different. You just need:

```javascript
cy.getByTestID(TESTID_BREADCRUMB_LAST)
```

Reply:
good idea, i'll change this